model-optimizer/extensions/ops/gelu.py

```diff
@@ -24,11 +24,17 @@ class GeLUOP(Op):
 
     def __init__(self, graph: Graph, attrs: dict):
         mandatory_props = {
-            'type': __class__.op,
-            'op': __class__.op,
+            'type': self.op,
```

Comment:
Please, update copyright year in this file.

Reply:
Fixed